docs/manual/MAIN_CONTROLLER_SPECS.md

```diff
@@ -23,6 +23,7 @@
 |||
 |Speaker Power     | 55 WPC @ 4 Ω |
 |                  | 32 WPC @ 8 Ω |
+|                  | 23 WPC all-channel driven power |
```

Comment:
I'm not sure how you want to format this in here, but these measurements are for **<0.1% THD+N** and taken at **1kHz**. Both of those are important because they strongly influence the result. For example the much higher 10%THD (note the lack of +N which also makes this weaker) of the amp chip spec results in a _much_ higher output wattage, but when tested I had to clip the signal to get 10% THD.

Reply:
I'm putting this in front of you largely cuz I have no idea how to properly represent that data; this is not my area of expertise.

Could you provide a recommendation on how you would represent it?